crates/loans/src/types.rs

```diff
@@ -59,7 +59,7 @@ impl<T: Config> AccountLiquidity<T> {
 #[derive(Encode, Decode, Eq, PartialEq, Copy, Clone, RuntimeDebug, Default, TypeInfo)]
 pub struct BorrowSnapshot<Balance> {
     /// Principal Total balance (with accrued interest), after applying the most recent balance-changing action
-    pub principal: Balance,
+    pub principal: Balance, // amount of underlying borrowed, to be paid back eventually
```

Comment:
Can move this comment to the doc comment on the line above

Reply:
I really did not understand the line above fwiw, but yea I'll delete